prometheus-exporters/ucs-exporter/values.yaml

```diff
@@ -0,0 +1,26 @@
+ucs:
+   enabled: false
+   image:
+      name: ucs-exporter
+      tag: DEFINED-IN-REGION-SECRETS
+   port_number: 9876
+   alerts:
+      enabled: false
+      prometheus: DEFINED-IN-REGION-SECRETS
+   user: DEFINED-IN-REGION-SECRETS
+   master_password: DEFINED-IN-REGION-SECRETS
```

Comment:
since your mapping is to .Values.global.master_password, this is never used and can go.